install-cockroachdb.md

```diff
@@ -99,7 +99,7 @@ $(document).ready(function(){
       <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="mac-binary-step2"><span class="gp" data-eventcategory="mac-binary-step2">$ </span>tar xfz cockroach-latest.darwin-10.9-amd64.tgz</code></pre></div>
     </li>
     <li>
-      <p>Move the binary into your <code>PATH</code> so it's easy to execute <a href="cockroach-commands.html">cockroach commands</a> from any shell:</p>
+      <p>Move the binary onto your <code>PATH</code> so it's easy to execute <a href="cockroach-commands.html">cockroach commands</a> from any shell:</p>
```

Comment:
"Into" reads more naturally to me than "onto" here. I see it as short for "move the binary into a directory on/in your PATH" - files are always "in" directories, while directories can be either "on" or "in" the PATH.